raw_data/preprocessed/africa/eswatini_english.txt

```diff
@@ -0,0 +1,291 @@
+
+Defences
+(70) [1] Except as otherwise specified in this Act, in any prosecution under this Act it is a defence for the person charged to prove that they took all reasonable precautions and exercised all such diligence to prevent the commission of the offence as ought to have been exercised by that person having regard to all the circumstance. [2] It is not a defence to a charge under this Act that the defendant was rendered unable to comply by an act or omission of the defendant that would have constituted a contravention of this Act.
+
+Continuing offences and penalties
+(71) [1] Except as otherwise specified in this Act or the regulations, every person who is convicted of an offence that is a continuing one, shall in addition to the penalty prescribed for the offence, be liable upon conviction to a further fine for each day or part of a day on which the offence continues, which: [a] in the case of an individual, shall not exceed Emalangeni [100] per day on a first conviction, and not more than Emalangeni [500] per day on each subsequent conviction; and [b] in the case of a legal person, shall not exceed Emalangeni [5,000] per day on a first conviction, and not more than Emalangeni [10,000] per day on each subsequent conviction. [2] Every natural person convicted of a contravention of section 34 or section 41[1] is liable upon conviction, for each day or part of a day on which the offence occurs or continues, to a fine of not exceeding Emalangeni [500] on a first conviction, and not less than Emalangeni [100], and not more than Emalangeni [20,000], or to a term of imprisonment of not more than [2 years], or to both, on each subsequent conviction. [3] Every body corporate convicted of a contravention of section 34 or section 41[1] is liable upon conviction, for each day or part of a day on which the offence occurs or continues, to a fine of not exceeding Emalangeni [10,000] on a first conviction, and not less than Emalangeni [1,000] and not exceeding Emalangeni [50,000], and to winding up, or to both, on each subsequent conviction. [4] A person who is found guilty of an offence under this Act for which no other penalty is provided shall, on conviction, be liable to imprisonment to a term not exceeding six months or to a fine not exceeding [5000] Emalangeni or to both.
```

Comment:
Please remove all of the square brackets around the Emalangeni quantities. Also, please replace `[2 years]` with `2 years`. Furthermore, please replace `5000` with `5,000`.